public/features.jade

```diff
@@ -3,28 +3,28 @@ div
     h2(class="text-headline") Cross Platform
     div(class="feature-row")
       div(class="feature")
-        h3 Progressive Web Apps
-        p(class="text-body") Use modern web platform capabilities to deliver app-like experiences.  High performance, offline and zero-step installation.
+        h3 Progressive
```

Comment:
Ah. I must have misunderstood the proofer's marks. This makes way more sense! :)